low-level/files/opt/seagate/sspl/setup/sspl_setup_init.py

```diff
@@ -64,7 +69,26 @@ def __init__(self):
         """initial variables and ConfStor setup."""
         self.role = None
         self.dp = True
-        Conf.load('sspl', f"yaml://{file_store_config_path}")
+        # Load sspl and global configs
+        Conf.load(SSPL_CONFIG_INDEX, sspl_config_path)
+        global_config_url = Conf.get(
+            SSPL_CONFIG_INDEX, "SYSTEM_INFORMATION>global_config_copy_url")
+        Conf.load(GLOBAL_CONFIG_INDEX, global_config_url)
+
+    def validate(self):
+        self.role = Conf.get(GLOBAL_CONFIG_INDEX, 'release>setup')
```

Comment:
> release>setup

already getting validated in sspl_config, why we need again here? I think we can skip here.

Reply:
yes, role validation is already happening with sspl_config.py.
Now, this validation is removed from sspl_setup_init.py

Reply:
Done.